Foundation/Process.swift

```diff
@@ -457,7 +457,7 @@ extension Task {
     }
 }
 
-public let NSTaskDidTerminateNotification: String = "NSTaskDidTerminateNotification"
+public let ProcessDidTerminateNotification: String = "ProcessDidTerminateNotification"
```

Comment:
There should be no `ProcessDidTerminateNotification` on Darwin: https://developer.apple.com/reference/foundation/process/1413681-didterminatenotification